File: src/sdk-reference/csharp/1/auth/get-current-user/snippets/get-current-user.cs
try {
  kuzzle.auth.login("local", @"{""username"":""foo"",""password"":""bar""}");
  User user = kuzzle.auth.getCurrentUser();

  Console.WriteLine("Successfully got current user");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.what());
}

File: src/sdk-reference/cpp/1/auth/get-current-user/snippets/get-current-user.cpp
try {
  kuzzle->auth->login("local", R"({"username":"foo","password":"bar"})");
  kuzzleio::User user = kuzzle->auth->getCurrentUser();

  std::cout << "Successfully got current user" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
